.prettierrc

```diff
@@ -15,5 +15,6 @@
   "semi": true,
   "tabWidth": 2,
   "trailingComma": "es5",
-  "vueIndentScriptAndStyle": false
+  "vueIndentScriptAndStyle": false,
+  "endOfLine": "auto"
```

Comment:
Let's discuss the eslint and code style rules before changing them. I think is better to come up with a cool set of rules from the group.

Reply:
Sorry, this change was inadvertently uploaded. I don't have a system unix based, for that reason I got an error in my code